cycle/cycle.env

```diff
@@ -102,6 +102,7 @@ ts_electrometer=0.8.0a1
 ts_fiberspectrograph=0.8.0
 ts_weatherforecast=0.3.1
 ts_gis=0.4.2a1
+robotframework_efd=develop
```

Comment:
yeah, so we usually want to have only tags/release numbers in cycle.env... any chance you can tag what you want to deploy instead of using develop? note, for instance that having develop branch here and trying to rebuild the container would not generate new artifacts unless you clean the container.. it also doesn't tell us which version of the package you are running.